roles/maintenance_32_apache/tasks/main.yml

```diff
@@ -0,0 +1,269 @@
+---
+- name: This task only serves as a template for the tasks below
+  ansible.builtin.set_fact:
+    ignoreme: &task
+      name: "{{ vars.taskid }}: {{ vars.name }}"
+      register: task
+      when:
+        - "vars.taskid not in maintenance_exclude_tasks"
+        - "maintenance_only is not defined or maintenance_only == vars.taskid"
+  vars:
+    taskid: ignore-me
+    name: bar
+
+# Include OS specific vars to get binary name: apache_binary_name, httpd or apache2
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Gather details, set vars: set apache binary based on OS"
+  ansible.builtin.include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_facts.os_family }}.yml"
+  changed_when: no
+
+# Get the exact binary path from the systemd service
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Gather details, set vars: query systemd service for binary name being used"
+  ansible.builtin.shell: "systemctl show -p ExecStart {{ apache_binary_name }} | grep 'ExecStart' | awk '{print $2}' | cut -d= -f2"
```

Comment:
I do this to use the exact binary systemd is using by default..

In RedHat.yml or Debian.yml, `apache_binary_name` is set to httpd or apache2, which will also be the systemd service name. (Maybe should rename it to include something related to systemd)

In the next task, I check if the systemd service was found, and if not, then the `apachectl` pointed to by the shell is defaulted to

This is what we agreed to do(to assume apache is managed through systemd, in case there are multiple installs of it present), LMK if this is not necessary in your experience